draft-tigress-requirements.md

```diff
@@ -180,7 +180,6 @@ end
 - (Req-Security) Solution should provide security of the provisioning data transferred (confidentiality, integrity and availability of provisioning information in transit).
 - (Req-NonCorelation) Transport protocol used to transfer provisioning information ( e.g. secure E2E transfer protocol or intermediary server) shall prevent from correlating users between exchanges or create a social graph of users involved into transfer.
 - (Req-NonIdentity) Intermediary server shall not be an arbiter of identity.
-- (Req-NonCollection) User identities shall not be collected, stored and used for purpose other then the credential transfer itself.
```

Comment:
Removing non-collection as requirement is good.

To me, this seems more suited for privacy considerations instead of security considerations. Should there be another consideration section called Privacy Consideration?

Reply:
We can call the paragraph "security and privacy"